TOR/Contribution.md

```diff
@@ -0,0 +1,46 @@
+# Contributing
+
+Thank you for your interest in contributing to the ETF project.
+
+Please first discuss the improvement you wish to make via
+[ETF Improvement Proposal (EIP) issue](https://github.com/etf-validator/governance/issues/new?template=etf-improvement-proposal--eip-.md)
+in our governance repository before creating a pull request.
+Pull Requests can be created for code changes that only fix bugs without an EIP,
```

Comment:
We should consistently say "bugs" (here) or "issues" (in the process below)

Reply:
I agree. I would prefer "bugs", because an EIP is also an GitHub "issue".